Documentation/installation/k8s-install-migration.rst

```diff
@@ -194,6 +194,35 @@ Preparation
         $ cat values-initial.yaml
 
 
+For example, Calico has several modes of operation to decide which network interface to use by default on nodes that it manages. This is configured using the Tigera Operator's ``spec.calicoNetwork.nodeAddressAutodetectionV4`` (and respectively ``nodeAddressAutodetectionV6`` for IPv6) parameter.
+By default, the parameter is set to ``firstFound: true``, which uses the first detected network interface on the node. Check this value with:
+
+.. code-block:: shell-session
+
+      kubectl get installations.operator.tigera.io default \
+  -o jsonpath='{.spec.calicoNetwork.nodeAddressAutodetectionV4}{"\n"}'
+
+5. Configure your existing CNI to avoid modifying Cilium-managed traffic.
```

Comment:
This also needs to be indented to match the indentation for "4." above. Subsequent paragraphs need to be indented to match the start of "Configure ...".

Reply:
This whole guide should have previously been written using `#.` instead of a number, but since we're using numbers for now.... Please double-check the rest of the document. I can see on line 226 that there is another `5.`. All subsequent numbers need to be updated if you're adding a new step here.